app/controllers/github_webhooks_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class GithubWebhooksController < ApplicationController
+  require 'faker'
+
+  def webhook
```

Comment:
This endpoint is open to spoofing, you aren't doing any checks to confirm that the request you're receiving is actually coming from Github, so anyone that knows this route and the format of Github's webhooks can fill your app with bad data. Github supports using a shared secret to handle exactly this type of validation, I'd like to see that implemented here: https://developer.github.com/webhooks/securing/

Reply:
Brilliant! I think it would make sense to store the secret in the rails credentials, which would allow to easily cut off any old webhooks by changing the secret in the credentials.